schema/dex/view_trades.sql

```diff
@@ -419,6 +419,24 @@ FROM (
         NULL::integer[] AS trace_address,
         evt_index AS evt_index
     FROM hydroprotocol."Margin_evt_Match"
+
+    UNION
+
+    -- Gnosis Protocol
+    SELECT
+        block_time,
+        'Gnosis Protocol' as project,
+        '1' as version,
+        trader_hex as trader_a,
+        NULL::bytea AS trader_b,
+        sell_amount_atoms as token_a_amount_raw,
+        buy_amount_atoms as token_b_amount_raw,
+        sell_token as token_a_address,
+        buy_token as token_b_address,
+        tx_hash,
```

Comment:
There should be a `exchange_contract_address`-value between `buy_token` and `tx_hash`

Reply:
Done!